documentation/components/JsonSchema.rst

```diff
@@ -146,6 +146,7 @@ Other options are available to customize the generated code:
    (``null``). By default it is enabled.
  * ``skip-required-fields``: If your model has required fields, this option allows you to skip the required behavior
    that forces them to be present during denormalization. By default it is disabled.
+ * ``validation``: Will enable validation following JSON Schema validation specification. By default it is disabled.
```

Comment:
Clearly require more text. A link to JSON Schema validation spec maybe and the fact we use Symfony Validator constraints. Maybe even an example of a generated validator ?